Sources/Build/llbuild.swift

```diff
@@ -92,7 +100,7 @@ public struct LLBuildManifestGenerator {
     }
 
     /// Create link command for products.
-    private func createLinkCommand(_ buildProduct: ProductBuildDescription) -> Command {
+    private func createLinkTarget(_ buildProduct: ProductBuildDescription) -> Target {
```

Comment:
I think the number of llbuild target should match the number of targets in the package graph. The executable targets will contain two commands (compile and link) and library targets will just contain one compile command. Is that how it works with this patch?